app/src/main/java/com/example/firstapp/AuthActivity.kt

```diff
@@ -0,0 +1,51 @@
+package com.example.firstapp
+
+import android.app.ActivityOptions
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Button
+import android.widget.EditText
+import androidx.appcompat.app.AppCompatActivity
+
+class AuthActivity : AppCompatActivity() {
+
+    private val preferenceManager by lazy { SharedPreferenceManager(this) }
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.auth_activity)
+//        auto login
+//       if (preferenceManager.getLogin().isNotBlank()) {
+//            val email = preferenceManager.getLogin()
+//            startMainActivity(email)
+//             finish()
+//            return
+//        }
+        val emailEditText: EditText = findViewById(R.id.emailEditText)
```

Comment:
use viewbidng instead of findViewById
https://developer.android.com/topic/libraries/view-binding